src/styles/main.scss

```diff
@@ -1,3 +1,36 @@
 body {
   margin: 0;
 }
+
+@import "./days/calendar.scss";
+@import "./variables/variables.scss";
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@for $day from 1 through 31 {
+  .calendar :nth-child(n + #{$day})::before {
+    content: "#{$day}" ;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
+}
+
+@each $day, $col in $days {
+  .calendar--start-day-#{$day} {
+    .calendar__day:first-child {
+      margin-left: ($day-size * $col) + $col * 1px;
+    }
+  }
+}
+
+@each $length in 28,29,30,31 {
+  .calendar--month-length-#{$length} {
+    .calendar__day:nth-child(n+#{$length + 1}) {
+      display: none;
+    }
+  }
+}
```

Comment:
you can move this styles to calendar file

Reply:
done